README.md

```diff
@@ -9,9 +9,7 @@ Run the latest version of the [Elastic stack][elk-stack] with Docker and Docker
 It gives you the ability to analyze any data set by using the searching/aggregation capabilities of Elasticsearch and
 the visualization power of Kibana.
 
-> :information_source: The Docker images backing this stack include [Stack Features][stack-features] (formerly X-Pack)
```

Comment:
Can we please stick to the current formatting for consistency? (line width = 120)

Reply:
Fixed.